applications/asset_tracker_v2/tests/json_common/src/main.c

```diff
@@ -800,7 +802,9 @@ static void test_decode_configuration_data(void)
 	zassert_equal(120, data.active_wait_timeout, "Configuration is wrong");
 	zassert_equal(120, data.movement_resolution, "Configuration is wrong");
 	zassert_equal(3600, data.movement_timeout, "Configuration is wrong");
-	zassert_equal(2, data.accelerometer_threshold, "Configuration is wrong");
+	zassert_equal(10, data.accelerometer_activity_threshold, "Configuration is wrong");
+	zassert_equal(5, data.accelerometer_inactivity_threshold, "Configuration is wrong");
+	zassert_equal(80, data.accelerometer_inactivity_timeout, "Configuration is wrong");
```

Comment:
Missing unit tests for the new parameters.

Reply:
should be added now